sound/soc/sof/intel/nvl.c

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// Copyright(c) 2025 Intel Corporation
+
+/*
+ * Hardware interface for audio DSP on PantherLake.
```

Comment:
For Nova Lake.

Reply:
On the other hand other files (again) have the name written w/o the space.